src/javascript/binary/base/page.js

```diff
@@ -854,6 +854,20 @@ Contents.prototype = {
             $('.unbind_later').off();
         }
     },
+    has_mlt_mf_enabled: function() {
```

Comment:
I don't like this kind of code. MLT and MF are broker codes. Business-logic should depend on the *landing company* not the broker code. Do we not have the landing company easily available in the front-end?